TesseractCore/Importing.swift

```diff
@@ -0,0 +1,62 @@
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
+
+private func nub<T: Equatable>(list: [T]) -> [T] {
+    var unique: [T] = []
+    for item in list {
+        if !contains(unique, item) {
+            unique.append(item)
+        }
+    }
+    return unique
```

Comment:
Might be better to use a `Set` instead of doing all these linear searches both when uniquing and then again when searching.